exporter/prometheusremotewriteexporter/config.go

```diff
@@ -50,6 +50,12 @@ type Config struct {
 
 	// TargetInfo allows customizing the target_info metric
 	TargetInfo *TargetInfo `mapstructure:"target_info,omitempty"`
+
+	ExportCreatedMetric *ExportCreatedMetric `mapstructure:"export_created_metric,omitempty"`
```

Comment:
export_created_metric sounds like it should be a bool, rather than a nested struct.  Consider `created_metric` if you are going to nest `enabled` within it.